uktrade/data-hub-frontend
# Company activity feed label styling changes

## Description of change
**All the code coverage errors are coming from the existing code that is now wrapped in the `CheckUserFeatureFlag` component. This old code is going to be removed as soon as the new code is given the sign off so no extra tests have been written for the existing code.**

This PR:
- Updates the styling for the Company Activity Feed activity labels. 
- Adds in a new user feature flag for these changes (`user-company-activities`)

Other additions to the card will be made in subsequent PRs.  

It looks like there are more changes than there actually are because I've had to wrap a lot of stuff in the `CheckUserFeatureFlag` component and add sandbox fixtures.
## Test instructions

- In Django dev API, add the user feature flag `user-company-activities` to your adviser profile and point your frontend towards the dev API.
- Go to http://localhost:3000/companies/375094ac-f79a-43e5-9c88-059a7caa17f0
- On the Activity Types dropdown, click on 'All Data Hub & external activity'.
- You should be able to see the new label changes for HMRC, Companies House Company, Companies House Accounts, Investment Project and OMIS.
- For the maxemail label change go to http://localhost:3000/companies/6df487c5-7c75-4672-8907-f74b49e6c635 and do the same as above.

## Screenshots
### Before
<img width="1007" alt="image" src="https://user-images.githubusercontent.com/70902973/185196230-f667572f-e390-4427-8666-37942119c483.png">

### After
<img width="1034" alt="image" src="https://user-images.githubusercontent.com/70902973/185196501-9ab2084f-351d-4fcd-86e6-68bd797f3bf9.png">

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
